.github/workflows/testingfarm-unit.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Testing farm go unit tests
```

Comment:
This duplication is sad, we should look into something like https://github.com/actions/starter-workflows/issues/245#issuecomment-1445128447 here too.